### PR TITLE
PSD-270: add sector component and some query functionality

### PR DESCRIPTION
**Pull request by go-bears**
Ticket:
https://kiva.atlassian.net/browse/PSD-270

What does this do:
loads sector component with left aligned checkboxes
writes functions that validate sector input, builds sector filter
changes loan display with updated queries from FLSS

How to check
`npm run unit -- flssUtil` runs the data functions that pull from `sector` and 'gender' props

Can spin up page
Sector checkboxes are visible but not yet functional (@emuvente knows they're not working)
page will mount with `US` loans
press "search" button on page -> loans display will change to `['Food' and 'Education']` sector loans 
press "reset filters" button on page -> loans filters are cleared and all fundraising loans will be display

Screen Shot 2021-09-22 at 11 36 09 AM

And the buttons unclick :)
Screen Shot 2021-09-22 at 12 44 47 PM

Also can watch console.log for filter updates:

Screen Shot 2021-09-22 at 11 44 17 AM

### CODE REVIEW

File: src/pages/Lend/FilterAlpha/LendFilterAlpha.vue [hunk 1]
```diff
@@ -178,12 +177,15 @@ export default {
...
 			this.loanQueryFilters = {};
-			this.runQuery();
+			this.runQuery(this.loanQueryFilters);
 		},
-		runQuery() {
-			console.log('filters into fetchData:', this.loanQueryFilters);
-			fetchData(this.loanQueryFilters, this.apollo).then(flssData => {
+		runQuery(loanQueryFilters) {
+			console.log('filters into runQuery:', loanQueryFilters);
+			fetchData(loanQueryFilters, this.apollo).then(flssData => {
```
Comment (mcstover) [inline src/pages/Lend/FilterAlpha/LendFilterAlpha.vue:188]: Why the change here...was calling `fetchData(this.loanQueryFilters, ...` not working?

**Review by mcstover — APPROVED** [review]

File: src/pages/Lend/FilterAlpha/LendFilterAlpha.vue (hunk 1 above)
Comment (go-bears) [reply src/pages/Lend/FilterAlpha/LendFilterAlpha.vue:188]: it was working, and then I thought I should be explicit about which query filters were going in at which step and having it as a param would maybe let me pull out run query to the utils.  I'm happy to change it back. :)

Comment (mcstover) [reply src/pages/Lend/FilterAlpha/LendFilterAlpha.vue:188]: That makes sense. No change necessary
